analyzers/tests/SonarAnalyzer.UnitTest/LiveVariableAnalysis/RoslynLiveVariableAnalysisTest.cs

```diff
@@ -864,6 +867,49 @@ public class Sample
             context.ValidateEntry(new Captured("intParameter"));
         }
 
+        [TestMethod]
+        public void PropertyWithWriteOnly()
+        {
+            var code = @"
+public class Sample
+{
+    private int target;
```

Comment:
Here we don't need the assignment, right? So it can be just `property` and not `target`

Reply:
Or just `Property => 42` and register for the literal